.github/dependabot.yml

```diff
@@ -8,83 +8,107 @@ updates:
     schedule:
       interval: "daily"
     ignore:
       - dependency-name: "tanzu-framework"
 
   - package-ecosystem: "gomod"
+    # TODO: Disable for now, need to review ignore list and reenable later.
+    open-pull-requests-limit: 0
     directory: "test/"
     schedule:
       interval: "daily"
     ignore:
       - dependency-name: "tanzu-framework"
 
   - package-ecosystem: "gomod"
+    # TODO: Disable for now, need to review ignore list and reenable later.
+    open-pull-requests-limit: 0
     directory: "hack/asset"
     schedule:
       interval: "daily"
     ignore:
       - dependency-name: "tanzu-framework"
 
   - package-ecosystem: "gomod"
+    # TODO: Disable for now, need to review ignore list and reenable later.
+    open-pull-requests-limit: 0
     directory: "hack/tools"
     schedule:
       interval: "daily"
     ignore:
       - dependency-name: "tanzu-framework"
 
   - package-ecosystem: "gomod"
+    # TODO: Disable for now, need to review ignore list and reenable later.
+    open-pull-requests-limit: 0
     directory: "hack/tagger"
     schedule:
       interval: "daily"
     ignore:
       - dependency-name: "tanzu-framework"
 
   - package-ecosystem: "gomod"
+    # TODO: Disable for now, need to review ignore list and reenable later.
+    open-pull-requests-limit: 0
     directory: "hack/packages"
     schedule:
       interval: "daily"
     ignore:
       - dependency-name: "tanzu-framework"
 
   - package-ecosystem: "gomod"
+    # TODO: Disable for now, need to review ignore list and reenable later.
+    open-pull-requests-limit: 0
     directory: "hack/dailybuild"
     schedule:
       interval: "daily"
     ignore:
       - dependency-name: "tanzu-framework"
 
   - package-ecosystem: "gomod"
+    # TODO: Disable for now, need to review ignore list and reenable later.
+    open-pull-requests-limit: 0
     directory: "hack/imagelinter"
     schedule:
       interval: "daily"
     ignore:
       - dependency-name: "tanzu-framework"
 
   - package-ecosystem: "gomod"
+    # TODO: Disable for now, need to review ignore list and reenable later.
+    open-pull-requests-limit: 0
     directory: "hack/runner/webhook"
     schedule:
       interval: "daily"
     ignore:
       - dependency-name: "tanzu-framework"
 
   - package-ecosystem: "gomod"
+    # TODO: Disable for now, need to review ignore list and reenable later.
+    open-pull-requests-limit: 0
     directory: "hack/release/release"
     schedule:
       interval: "daily"
     ignore:
       - dependency-name: "tanzu-framework"
 
   - package-ecosystem: "gomod"
+    # TODO: Disable for now, need to review ignore list and reenable later.
+    open-pull-requests-limit: 0
     directory: "hack/workflows/packages"
     schedule:
       interval: "daily"
     ignore:
       - dependency-name: "tanzu-framework"
 
   - package-ecosystem: "gomod"
+    # TODO: Disable for now, need to review ignore list and reenable later.
+    open-pull-requests-limit: 0
```

Comment:
i'm ok with this one being on.

Reply:
FYI - This is the shared library for testing between the different packages; I agree, I think it makes sense to keep dependabot on for it but seems to be a "gray area" for ownership between our bits and the `addons/packages` bits package owners own.